smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -22,6 +25,11 @@ contract MixinKeys is
 
   // Called when the Lock owner expires a user's Key
   event ExpireKey(uint indexed tokenId);
+  event ExpirationChanged(
```

Comment:
Other event needs to be removed.

And should we put it in the interface?  We've been inconsistent about that.
To avoid redundancy - maybe we ONLY put events in the interface.

Reply:
Actually this is something I've been thinking as well. I do think events should ONLY be defined in the interface. I will move this one over.

Reply:
> Other event needs to be removed.
are you referring to the `ExpireKey` event? That one has to stay... `ExpirationChanged` is just a renaming of `TimestampChanged`

Reply:
https://github.com/unlock-protocol/unlock/blob/n44o-issue-5728-2/smart-contracts/contracts/mixins/MixinTransfer.sol#L33

Reply:
Ahh, yes. Thx. 
Regarding putting all the events in the interface, I guess we'll need to import & inherit the interface in pretty much all the mixins. Should be fine, as IPublicLock is abstract.

Reply:
@NickCuso I'm thinking that  moving all event definitions into the interface could be a separate PR. I'm pretty sure that when inheriting the interface in each Mixin, we're going to hit a bunch of circular dependency errors.
One possible (cleaner?) approach would be to split up our current abstract `IPublicLock` into "pure" interfaces (one for each mixin) each inherited by its respective mixin. `IPublicLock` could inherit each of these small interfaces as well, and then PublicLock still only has to inherit the single interface. I will create an issue around these ideas if you think its worth pursueing?

Reply:
New issue here: https://app.zenhub.com/workspaces/unlock-5bfd5c894b5806bc2bf85d03/issues/unlock-protocol/unlock/5866